src/backend/gporca/server/src/unittest/CTestUtils.cpp

```diff
@@ -3827,13 +3827,14 @@ CTestUtils::CreateGenericDatum(CMemoryPool *mp, CMDAccessor *md_accessor,
 		CDXLUtils::DecodeByteArrayFromString(mp, pstrEncodedValue, &ulbaSize);
 
 	CDXLDatumGeneric *dxl_datum = NULL;
-	if (CMDTypeGenericGPDB::IsTimeRelatedType(mdid_type))
+	if (gpmd::CMDTypeGenericGPDB::IsTimeRelatedTypeMappableToDouble(mdid_type))
```

Comment:
Do we need an explicit `gpmd` namespace usage here?

Reply:
Removed. I supposed using of `gpmd` namespace is not defined in this module